### ISSUE TITLE
When I increase the accumulate_gradient_steps, can the batch_size also be increased accordingly?

### ISSUE THREAD

**Issue by joytianya** [young-geng/EasyLM#30]
if it is normal 
...
--train_dataset.json_dataset.batch_size=4
--optimizer.bf16_accumulate_gradient=True \
--optimizer.accumulate_gradient_steps=1 \
...

so is it right?
...
--train_dataset.json_dataset.batch_size=8
--optimizer.bf16_accumulate_gradient=True \
--optimizer.accumulate_gradient_steps=2 \
...

**Comment by young-geng** [comment]
The effective batch size after accumulating the gradient is accumulate_gradient_steps * batch_size, so you don't need to change the batch_size
